Compos/Helpers/SqlReader.cs

```diff
@@ -1,29 +1,27 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Data;
+using System.Diagnostics;
 using System.Globalization;
 using System.IO;
+using System.Linq;
 using System.Reflection;
 using Microsoft.Data.Sqlite;
 
+
 namespace ComposAPI.Helpers {
   /// <summary>
   /// Class containing functions to interface with SQLite db files.
   /// </summary>
   public class SqlReader : MarshalByRefObject {
     public static SqlReader Instance => lazy.Value;
+   // public static Assembly[] assemblies = AppDomain.CurrentDomain.GetAssemblies();
     private static readonly Lazy<SqlReader> lazy = new Lazy<SqlReader>(() => Initialize());
-
+   // private static assemblies = AppDomain.GetAssemblies();
     public SqlReader() {
     }
 
-    public static SqlReader Initialize() {
-      string codeBase = Assembly.GetCallingAssembly().CodeBase;
-      var uri = new UriBuilder(codeBase);
-      string codeBasePath = Path.GetDirectoryName(Uri.UnescapeDataString(uri.Path));
-
-      Assembly.LoadFile(codeBasePath + @"\Microsoft.Data.Sqlite.dll");

```

Comment:
Is this for Rhino-Compute?

Reply:
@psarras Sqlite has been used to read section catalogue from C:\Program Files\Oasys\Compos 8.6

For me code is working without this else always throwing exception. SqLite has already been reference through Nuget and always there is project path from where It can load dll on need automatically. We we need load DLL manually?

Reply:
![image](https://github.com/user-attachments/assets/591d525d-cb6f-4c12-8bf2-dd5690ec98d1)
Let's wait for Ali to Reply

Reply:
Ali cannot pin point something specific to this.

Reply:
It is related to this: `bugfix/COMPSGH-168-Reading-ComposGH-before-GsaGH-removes-GsaGH-s-CreateProfile-component` and https://ovearup.atlassian.net/browse/COMPSGH-168

Reply:
So it seems this was added to resolve a conflict between GSA.GH, Compos.GH and AdSec.GH, it seems that slowly they moved this class to Oasys.GH as a common class.

Reply:
However, removing this line along with the one found on GSA.GH doesn't produce any errors locally. On the other hand this is also located on the Oasys.GH and it will require more to remove and reference to all projects

Reply:
Ok, so I can also confirm that removing the Assembly.Load("sqlite.dll") everywhere doesn't mean there are conflicts locally. We need a final manual test within ArupCompute to confirm.

Reply:
However, let's remove this for now, as it is not used in ArupCompute. I believe this was set while different version of SQLite were used across the plugins.